### ISSUE TITLE
NTR: Asialotransferrin low in CSF compared to transferrin (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8472]
New term request Asialotransferrin low in CSF compared to transferrin (segal_110418210548): Desialotransferrin, tau protein; cerebrospinal fluid  UMLS: 1832741

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000447
- label: Decreased CSF asialotransferrin to transferrin ratio
- parents: [Abnormal CSF protein concentration (HP:0025456)](https://hpo.jax.org/app/browse/term/HP:0025456)
- pmid: PMID:18519871
- def: The concentration of asialotransferrin relative to that of transferrin in the cerebrospinal fluid (CSF) is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8472
- synonyms:
